designs/developer/operator-serverless-45/event-sources.md

```diff
@@ -0,0 +1,48 @@
+---
+parent: Developer
```

Comment:
Is this intended to stick around in the documentation? The image links are broken if so :(

Reply:
This will get updated in a follow on PR

Reply:
Sounds good!